RFC/src/RFC-0304_VNCommittees.md

```diff
@@ -1,6 +1,95 @@
+technological merits of the potential system outlined herein.
+
+## Goals
+
+This document will describe the process an [Asset Issuer] (AI) will go through in order to select the committee of [Validator Node]s
+(VNs) that will serve a given [Digital Asset] (DA).
+
+## Related RFCs
+* [RFC-0311: Digital Asset templates](RFC-0311_AssetTemplates.md)
+* [RFC-0302: Validator Nodes](RFC-0302_ValidatorNodes.md)
+* [RFC-0341: Asset Registration](RFC-0341_AssetRegistration.md)
+
+## Description
+
+### Abstract
+[Digital Asset]s (DAs) are managed by [committee]s of nodes called [Validator Node]s (VNs), as described in [RFC-0300](RFC-0300_DAN.md) and [RFC-0302](RFC-0302_ValidatorNodes.md). During the asset creation process, described in [RFC-0341](RFC-0341_AssetRegistration.md), the [Asset Issuer] (AI) needs to select a committee of VNs to manage their asset. This process consists of the following steps:
+
+1. Candidate VNs need to be nominated to be considered for selection by the AI.
+2. The AI must employ a [CommitteeSelectionStrategy] to select VNs from the set of nominated candidates.
+3. The AI makes an offers of committee membership to the selected VNs.
+4. Selected VNs may accept the offer to become part of the committee by posted the required [AssetCollateral].
+
+### Nomination
+The first step in assembling a committee is to nominate candidate VNs. As described in [RFC-0311](RFC-0311_AssetTemplates.md) an asset can be created with two possible `committee_modes`: `CREATOR_NOMINATION` or `PUBLIC_NOMINATION`.
+
+In `CREATOR_NOMINATION` mode the AI nominates candidate committee members directly. The AI will have a list of permissioned [Trusted Node]s that they want to act as the committee. The AI will contact the candidate VNs directly to inform them of their nomination.
+
+In `PUBLIC_NOMINATION` mode the AI does not have a list of [Trusted Node]s and wants to source unknown VNs from the network. In this case the AI broadcasts a public call for nomination to the Tari network using the peer-to-peer messaging protocol described in [RFC-0172](RFC-0172_PeerToPeerMessagingProtocol.md). This call for nomination contains all the details of the asset and VNs that want to participate will then nominate themselves by contacting the AI.
+
+### Selection
+Once the AI has received a list of nominated VNs it must make a selection, assuming enough VNs were nominated to populate the committee. In `CREATOR_NOMINATION` mode this selection should trivial as the AI directly nominated [Trusted Node]s.
+
+In `PUBLIC_NOMINATION` mode the AI will employ some [CommitteeSelectionStrategy] in order to select the committee. This strategy might aim for a perfectly random selection or perhaps it will consider some metrics about the candidate VNs such as the length of their VN registrations which might indicate that they are reliable and have not been black listed for poor or malicious performance.
```

Comment:
```suggestion
In `PUBLIC_NOMINATION` mode the AI will employ some [CommitteeSelectionStrategy] in order to select the committee. This strategy might aim for a perfectly random selection or perhaps it will consider some metrics about the candidate VNs such as the length of their VN registrations which might indicate that they are reliable and have not been blacklisted for poor or malicious performance.
```